### DISCUSSION TITLE
Chart Extension: Second line graph

### DISCUSSION THREAD

**Discussion by Sigurdur42** [Megabit/Blazorise#2453] [Q&A]
Hi,

I am new to Blazorize but like it so far. I want to use the chart extension - but I want to display 2 lines with different values below each other. The value range would fit in the same graph.
How can I set this up or is it not possible yet?
Thx
Michael

**Comment by stsrki** [comment]
It's very simple. Dataset is an array so you only need to add another data and it should work 

![image](https://user-images.githubusercontent.com/900302/120917450-aed98700-c6af-11eb-8131-2f62d425b341.png)

If you follow our docs or demo code, just add 

```cs
await chart.AddLabelsDatasetsAndUpdate( Labels, getDataSet() , getDataSet2() );
```
